### DISCUSSION TITLE
sat export / cut drillings option?

### DISCUSSION THREAD

**Discussion by hgeglo** [cwapi3d/cwapi3dpython#207] [Q&A]
Is it somehow possible to set the "cut drillings" option for the export_sat_file() function?

**Comment by Brunner246** [comment]
Hi,
This is not possible. 
At the moment there is only a function for exporting drill holes for `step`.
see https://docs.cadwork.com/projects/cwapi3dpython/en/latest/documentation/file_controller/#src.file_controller.export_step_file_extrude_drillings

i will add this request into our backlog.

**Comment by hgeglo** [comment]
Hi, 
The function "export_sat_file_cut_drillings" is available now and that's great, thanks. However it seems that some temporary elements are not deleted correctly from the model afterward? At least that's my impression: the function works well, but if i run a defective parts search after, I get quite a few "ghost" parts?

**Comment by Brunner246** [comment]
Hi @hgeglo 
Good catch!

This will be fixed from build >= `32.0.323`
